### PR TITLE
Update RoleNORManager.md

### PR DESCRIPTION
**Pull request by matentzn**
Replaces #2571

### CODE REVIEW

File: docs/roles/RoleNORManager.md [hunk 1]
```diff
@@ -98,3 +98,10 @@ If and only if the requested ontology passes the dashboard, i.e. no more "red" e
 When the ontology is accepted, the OBO NOR reviewer needs to make sure that the accepted ontology is removed from the OBO NOR Dashboard.
 
 Note 29.11.2022: There is a [work in progress](https://github.com/OBOFoundry/obo-nor.github.io/pull/23) which can deal with this step automatically, but the responsibility still stays with the OBO NOR Manager.
+
+### Maintaining the OBO Duty Rotation Table
+
+The OBO Duty Rotation Table is a Google Doc available [here](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit).
```
Comment (nlharris) [inline docs/roles/RoleNORManager.md:104]: mention that it is editable by everyone on the OBO Ops committee?

File: docs/roles/RoleNORManager.md [hunk 2]
```diff
@@ -98,3 +98,10 @@ If and only if the requested ontology passes the dashboard, i.e. no more "red" e
...
+When a new member is added to OBO Foundry, the OBO NOR reviewer should add their name to this table. Likewise, former members should be removed from the table. The soure of truth for OBO Foundry member is the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).
```
Comment (nlharris) [inline docs/roles/RoleNORManager.md:105]:
```suggestion
When a new member is added to OBO Foundry, the OBO NOR reviewer should add their name to this table. Likewise, former members should be removed from the table. The list of current OBO Foundry members is maintained in the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).
```

File: docs/roles/RoleNORManager.md [hunk 3]
```diff
@@ -98,3 +98,10 @@ If and only if the requested ontology passes the dashboard, i.e. no more "red" e
...
+
+In addition, when a new ontology has been added, it should be documented to the 'Ontology Review Rotation' sheet in the same document.
```
Comment (nlharris) [inline docs/roles/RoleNORManager.md:107]:
```suggestion
In addition, when a new ontology has been submitted, it should be documented in the 'Ontology Review Rotation' sheet in the same document.
```

File: docs/roles/RoleNORManager.md (hunk 1 above)
Comment (nlharris) [inline docs/roles/RoleNORManager.md:104]:
```suggestion
The OBO Duty Rotation Table is a Google spreadsheet available [here](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit).
```

**Review by nlharris — CHANGES_REQUESTED** [review]
Requesting minor edits

File: docs/roles/RoleNORManager.md [hunk 4]
```diff
@@ -98,3 +98,10 @@ If and only if the requested ontology passes the dashboard, i.e. no more "red" e
 When the ontology is accepted, the OBO NOR reviewer needs to make sure that the accepted ontology is removed from the OBO NOR Dashboard.
 
 Note 29.11.2022: There is a [work in progress](https://github.com/OBOFoundry/obo-nor.github.io/pull/23) which can deal with this step automatically, but the responsibility still stays with the OBO NOR Manager.
+
+### Maintaining the OBO Duty Rotation Table
+
+The OBO Duty Rotation Table is a Google spreadsheet available [here](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit).
+When a new member is added to OBO Foundry, the OBO NOR reviewer should add their name to this table. Likewise, former members should be removed from the table. The list of current OBO Foundry members is maintained in the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).
+
+In addition, when a new ontology has been submitted, it should be documented in the 'Ontology Review Rotation' sheet in the same document.
```
Comment (nlharris) [inline docs/roles/RoleNORManager.md:107]:
```suggestion
In addition, when a new ontology has been submitted, it should be added to the ['Ontology Reviewers' tab](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit?gid=1683009411#gid=1683009411) in the OBO Duty Rotation spreadsheet.
```

Comment (nlharris) [reply docs/roles/RoleNORManager.md:107]: Sorry, I missed this in my first edit.

**Review by nlharris — CHANGES_REQUESTED** [review]
Changes look fine. I found another sentence that was confusing and suggested edits.

**Comment by pfabry** [comment]
What was the problem with the previous PR ? Since it seemed to be on my side, I'd like to avoid repeating it.

**Comment by matentzn** [comment]
> What was the problem with the previous PR ? Since it seemed to be on my side, I'd like to avoid repeating it.

I am not sure, but it _looked_ to me like you were using a windows computer to save the documentation, causing _all_ lines to appear in the diff. Check the diff (files changed) in you PR to see how much red and green there is!

**Comment by pfabry** [comment]
> I am not sure, but it _looked_ to me like you were using a windows computer to save the documentation, causing _all_ lines to appear in the diff. Check the diff (files changed) in you PR to see how much red and green there is!

Indeed, I am using a Windows computer. If I'am converting the EOL characeter to Unix format, that should do the trick?

**Comment by matentzn** [comment]
I would hope so, but I can't promise it :)

**Comment by nlharris** [comment]
What's the next step here? Should Nico make the requested change?

File: docs/roles/RoleNORManager.md [hunk 5]
```diff
@@ -98,3 +98,10 @@ If and only if the requested ontology passes the dashboard, i.e. no more "red" e
 When the ontology is accepted, the OBO NOR reviewer needs to make sure that the accepted ontology is removed from the OBO NOR Dashboard.
 
 Note 29.11.2022: There is a [work in progress](https://github.com/OBOFoundry/obo-nor.github.io/pull/23) which can deal with this step automatically, but the responsibility still stays with the OBO NOR Manager.
+
+### Maintaining the OBO Duty Rotation Table
+
+The OBO Duty Rotation Table is a Google spreadsheet available [here](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit).
+When a new member is added to OBO Foundry, the OBO NOR reviewer should add their name to this table. Likewise, former members should be removed from the table. The list of current OBO Foundry members is maintained in the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).
```
Comment (nlharris) [inline docs/roles/RoleNORManager.md:105]:
```suggestion
When a new member is added to the OBO Operations Committee, the OBO NOR reviewer should add the new member's name to this table. Similarly, former members should be removed from the table. The list of current OBO Operations members is maintained in the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).
```

**Review by nlharris — CHANGES_REQUESTED** [review]

**Comment by nataled** [comment]
FWIW, I agree with Nomi's proposed change.

**Comment by pfabry** [comment]
> FWIW, I agree with Nomi's proposed change.

Me too.

**Comment by nlharris** [comment]
I assume I'm not supposed to re-review this?

**Comment by matentzn** [comment]
For QC purposes you have to approve!

**Review by nlharris — APPROVED** [review]

**Pull request merged**